### PR TITLE
Remove ReadWriteOncePod feature gate

### PR DESCRIPTION
**Pull request by chrishenzie**
This feature is GA and its feature gates will be removed in v1.31:
https://github.com/kubernetes/kubernetes/pull/124329

Following instructions in:
https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates

/assign sftim
/assign carlory

### CODE REVIEW

**Review by shannonxtreme — COMMENTED** [review]
Seems okay to me

/lgtm

**Comment by tengqm** [comment]
Docs looks okay, but the upstream change is not yet merged.
Please add a link to the upstream PR.
/hold

**Comment by chrishenzie** [comment]
Upstream PR here: https://github.com/kubernetes/kubernetes/pull/124329, also added to description

**Comment by tengqm** [comment]
/approve
Change looks good.
Feel free to lift the hold when the upstream change is in.

File: content/en/docs/reference/command-line-tools-reference/feature-gates/read-write-once-pod.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ stages:
   - stage: stable
     defaultValue: true
     fromVersion: "1.29" 
+    toVersion: "1.31"
```
[the remark this answers is not in the document]
Comment (carlory) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/read-write-once-pod.md:20]:
```
toVersion: <Version>              # (Optional) The version until which the feature gate is available
```

[the remark this answers is not in the document]
Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/read-write-once-pod.md:20]: Right. It should be 1.30 if the gate is gone starting from 1.31.

[the remark this answers is not in the document]
Comment (chrishenzie) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/read-write-once-pod.md:20]: Oops, thanks for catching. Fixed.

**Comment by carlory** [comment]
/hold cancel

upstream pr is merged

**Pull request merged**